assets/js/main.js

```diff
@@ -466,7 +466,11 @@ function updateFissure() {
 }
 
 function updateNews() {
-  const {news} = worldState;
+  let {news} = worldState;
+  news = news.filter((article) => {
+    if ('en' in article.translations) { return true; }
```

Comment:
uhhhh, why using this instead of dot or how i did it? i thought the complaint was just because it wasn't the dot usage

Reply:
```js
let {news} = worldState;
news = news.filter(article => article.translations.en);
```
should be sufficient to check truthiness